src/Akka.Bootstrap.Docker/StringExtension.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Akka.Bootstrap.Docker
+{
+    public static class StringExtension
+    {
+        public const string NotInUnquotedText = "$\"{}[]:=,#`^?!@*&\\";
+        public const char NewLine = '\u000A';
```

Comment:
Shouldn't this be [`Environment.NewLine`](https://docs.microsoft.com/en-us/dotnet/api/system.environment.newline?view=net-5.0)?

Reply:
Nope, HOCON is very specific on what a newline is

Reply:
Anything else is considered as whitespace... oh wait, i forgot that this parser is not a hardened parser...

Reply:
I'll change that, give me a sec

Reply:
That value might actually be ok - tests passed on Linux and Windows. Add a test case to the suite that would hit this and see if it makes a difference